src/services/errors/interfaces/http-service-error-response.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface HttpServiceErrorResponseData {
+  code: number;
+  message: string;
+  arguments: string[];
+}
```

Comment:
Similar to what I commented here https://github.com/5afe/safe-client-gateway-nest/pull/11/files#r940172896

Maybe we can remove the `export` on this one since this mapping should only be known to the mapper itself what do you think?

Reply:
You mean having this interface into the `http-error-handler.ts` file? It's used on the unit tests as well :)

Reply:
Ah ok! I need to review how Typescript handles modules in general. For me this makes sense to be included here and in the tests but not in other modules.

Reply:
I agree, I think TS/JS is more permissive than other languages (like Java) with importing/exporting things from different places in the code. A lot of things fall in developer's responsibility and good practices, which sometimes is not so good 😄

Reply:
@fmrsabino I removed this class because since the last refactor it's redundant with `HttpExceptionPayload` 🙂